### PR TITLE
Fix user class

### PR DESCRIPTION
**Pull request by hapiben**

### CODE REVIEW

File: views/login.rb [hunk 1]
```diff
@@ -1,39 +1,35 @@
 module Omnigollum
   module Views
     class Login < Mustache
-      self.template_path = File.expand_path("../../templates", __FILE__)
+      self.template_path = File.expand_path('../templates', __dir__)
```
Comment (arr2036) [inline views/login.rb]: What's the justification for this??

**Comment by arr2036** [comment]
It's really quite obnoxious mixing in a bunch of code beautifier changes with a completely unrelated set of commits, and then not really describing what any of it does or what it's for...

File: views/login.rb (hunk 1 above)
Comment (hapiben) [reply views/login.rb]: Wrong PR. This is meant for the forked version.

**Comment by hapiben** [comment]
@arr2036 Agree. This is meant for the forked version for us. That's why I closed it.

**Pull request closed without merging**